site.yml

```
@@ -21,460 +21,460 @@ Documents:
+        SEO Description: This quickstart guide will show you how to install the client SDK and execute an example JavaScript program.
+        SEO Keywords: JavaScript, JavaScript API, npm install minio
 
       - JavaScript Client API Reference:
         Link: https://raw.githubusercontent.com/minio/minio-js/master/docs/API.md
         Slug: javascript-client-api-reference
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | JavaScript Client API Reference
+        SEO Description: This document contains the detailed example code for the JavaScript Client API
+        SEO Keywords: JavaScript Client, JavaScript API
 
       - Java Client Quickstart Guide:
         Link: https://raw.githubusercontent.com/minio/minio-java/master/README.md
         Slug: java-client-quickstart-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Java Client Quickstart Guide
+        SEO Description: The MinIO Java Client SDK provides simple APIs to access any Amazon S3 compatible object storage server.
+        SEO Keywords: Java Client SDK
 
       - Java Client API Reference:
         Link: https://raw.githubusercontent.com/minio/minio-java/master/docs/API.md
         Slug: java-client-api-reference
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Java Client API Reference
+        SEO Description: The MinIO Java Client SDK provides detailed code examples for the Java API
+        SEO Keywords: Java Client SDK, Java Client API
 
       - Python Client Quickstart Guide:
         Link: https://raw.githubusercontent.com/minio/minio-py/master/README.md
         Slug: python-client-quickstart-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Python Client Quickstart Guide
+        SEO Description: The MinIO Python SDK provides simple APIs to access any Amazon S3 compatible object storage server.
+        SEO Keywords: Python SDK, Python Client API
 
       - Python Client API Reference:
         Link: https://raw.githubusercontent.com/minio/minio-py/master/docs/API.md
         Slug: python-client-api-reference
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Python Client API Reference
+        SEO Description: The MinIO Python SDK provides detailed code examples for the Java API
+        SEO Keywords: Python SDK, Python Client API
 
       - Golang Client Quickstart Guide:
         Link: https://raw.githubusercontent.com/minio/minio-go/master/README.md
         Slug: golang-client-quickstart-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Golang Client Quickstart Guide
+        SEO Description: MinIO | Golang Client Quickstart Guide	The MinIO Go Client SDK provides simple APIs to access any Amazon S3 compatible object storage server.
```

I think there was some extra text left on the description here, we can do the following change:
```suggestion
        SEO Description: The MinIO Go Client SDK provides simple APIs to access any Amazon S3 compatible object storage server.
```

---

ok